.github/fabricbot/updateAreaPodConfigs.js

```diff
@@ -460,14 +461,20 @@ let areaPodConfig = {
             "columnName": "Triaged",
             "isOrgProject": true
           }
+        },
+        {
+          "name": "removeLabel",
+          "parameters": {
+            "label": "untriaged"
```

Comment:
I'm not sure we should remove `untriaged` if the issue has been marked `needs-author-action`. Perhaps we should make this a global rule unrelated to specific pods -- removing the `untriaged` label if an issue is added to a milestone or closed.

Reply:
mm I kind of disagree here. We have been saying all this time that if an issue get's the label `needs-author-action` added then we consider it as triaged. Why would our `untriaged` label mean something different?

Reply:
I could see an argument for doing something repo-wide where getting a milestone applied, closing the issue, or adding `needs-author-action` or `api-ready-for-review` would automatically remove `untriaged`. But I do think we'd want it to be any of those 4 actions as opposed to limiting it to the milestone/closed actions.

I thought earlier today I'd want to propose we go ahead and do this repo-wide, but after some more discussions today, I think it's best for us to pilot this automation within the libraries area pods for now. If the automation of removing `untriaged` works well for us, we could then generalize it.

Reply:
> We have been saying all this time that if an issue get's the label needs-author-action added then we consider it as triaged.

From a triage board perspective, adding `needs-author-action` removes the issue from the actionable "Needs Triage" basket but otherwise it is still undergoing triage.

Reply:
> From a triage board perspective, adding needs-author-action removes the issue from the actionable "Needs Triage" basket but otherwise it is still undergoing triage.

Leaving https://github.com/dotnet/runtime/labels/untriaged on the issue while also having https://github.com/dotnet/runtime/labels/needs-author-action would be a disparity between what needs triage and what's actionable. I'm in favor of removing https://github.com/dotnet/runtime/labels/untriaged for these, and moving them into our _Triaged_ column to resolve that disparity. When the additional info is collected, the issue will come back into the _Needs Triage_ column, prompting us to conduct further triage.